omnigan/generator.py

```diff
@@ -334,7 +347,8 @@ def concat_bit_to_seg(self, seg):
         return torch.cat([bit, seg.to(torch.float32)], dim=1)
 
     def forward(self, x, seg):
-        seg = self.concat_bit_to_seg(seg)
+        if self.use_bit_conditioning:
```

Comment:
if using bit conditioning, add bit to segmentation as conditionning data otherwise as it's just a standard standalone model just use `seg` as conditionning data